README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
The `README` file is generated using an internal `Grunt` task. Can you please apply the update to the file [`docs/qunit-overview.md`](https://github.com/gruntjs/grunt-contrib-qunit/blob/master/docs/qunit-overview.md), run `grunt` to generate the new `README` and then commit the updated files.